File: src/main/java/de/itsTyrion/flusenbot/command/commands/Soon.java
package de.itsTyrion.flusenbot.command.commands;

import com.pengrad.telegrambot.model.Chat;
import com.pengrad.telegrambot.model.User;
import de.itsTyrion.flusenbot.command.Command;
import de.itsTyrion.flusenbot.command.CommandInfo;

@CommandInfo(name = "sOoN")
public class Soon extends Command {

    @Override
    protected boolean execute(String[] args, Chat chat, User user) {
        reply(chat, "Soon™", 5);
        return true;
    }
}

File: src/main/java/de/itsTyrion/flusenbot/command/commands/Soon.kt
package de.itsTyrion.flusenbot.command.commands

import com.pengrad.telegrambot.model.Message
import de.itsTyrion.flusenbot.command.Command

class Soon : Command("sOoN") {

    override fun execute(args: Array<String>, msg: Message): Boolean {
        reply(msg.chat(), "Soon™", 5)
        return true
    }
}